src/main/java/io/xpring/xrpl/model/XRPPayment.java

```diff
@@ -38,35 +38,29 @@
    *
    * @return An {@link Integer} containing the tag that identifies the reason for the payment.
```

Comment:
update this for Optional

Reply:
Updated javadocs with Optionals everywhere.